go-controller/pkg/ovn/loadbalancer.go

```diff
@@ -267,12 +267,13 @@ func (ovn *Controller) createLoadBalancerRejectACL(lb string, sourceIP string, s
 }
 
 func (ovn *Controller) deleteLoadBalancerRejectACL(lb, vip string) {
-	acl, _ := ovn.getServiceLBInfo(lb, vip)
```

Comment:
with this change seems that ` ovn.getServiceLBInfo(lb, vip)` does no longer needs to return the acl.
Is it always better to generate it from the lb,vip?

Reply:
I modified this to still try to use the cache, and if it finds the cache is invalid then last resort is try to query OVN. So we still need this. See upcoming change.